### ISSUE TITLE
The maximize window cannot be restored from the taskbar

### ISSUE THREAD

**Issue by arinayarina** [love2d/love#1716]
I am using `love.window.maximize` to stretch the window to full screen. When I minimize the window, I can no longer maximize it back. There is an animation that it unfolds, but it immediately disappears as soon as the animation ends.
If the window is moved from its original position, it unfolds normally.
windows 10. love 11.4

**Comment by alex-boulay** [comment]
it looks like a bug, if u want a quickfix :
Add or change your love.focus function
```
function love.focus(t)
  if t and love.window.isMaximized() then
    love.window.restore()
    love.window.maximize()
  end
end
```

**Comment by slime73** [comment]
This only seems to happen in non-resizable windows - so another workaround is to make it resizable.

**Comment by slime73** [comment]
`love.window.maximize` actually has no effect on some other platforms (like macOS) if the window isn't set as resizable - I wonder if love's code should change to ignore it in that situation (along with documentation updates).

**Comment by slime73** [comment]
This was solved in the version of SDL used by love 11.5 (and newer) - it doesn't let you programmatically maximize a non-resizable window on Windows anymore.
